clients/venice-thin-client/src/main/java/com/linkedin/venice/client/stats/BasicClientStats.java

```diff
@@ -56,37 +63,71 @@ protected BasicClientStats(MetricsRepository metricsRepository, String storeName
         new TehutiUtils.SimpleRatioStat(successRequestKeyCountRate, requestKeyCountRate));
   }
 
+  protected boolean isMetaSystemStore() {
+    return isMetaSystemStore;
+  }
+
   private void recordRequest() {
+    if (isMetaSystemStore()) {
+      return;
+    }
     requestSensor.record();
   }
 
   public void recordHealthyRequest() {
+    if (isMetaSystemStore()) {
+      return;
+    }
     recordRequest();
     healthySensor.record();
   }
 
   public void recordUnhealthyRequest() {
+    if (isMetaSystemStore()) {
+      return;
+    }
     recordRequest();
     unhealthySensor.record();
   }
 
   public void recordHealthyLatency(double latency) {
+    if (isMetaSystemStore()) {
+      return;
+    }
     healthyRequestLatencySensor.record(latency);
   }
 
   public void recordRequestKeyCount(int keyCount) {
+    if (isMetaSystemStore()) {
+      return;
+    }
     requestKeyCountSensor.record(keyCount);
   }
 
   public void recordSuccessRequestKeyCount(int successKeyCount) {
+    if (isMetaSystemStore()) {
+      return;
+    }
     successRequestKeyCountSensor.record(successKeyCount);
   }
 
   protected final Rate getRequestRate() {
     return requestRate;
   }
 
-  protected final Rate getSuccessRequestKeyCountRate() {
-    return successRequestKeyCountRate;
+  @Override
+  protected Sensor registerSensor(String sensorName, MeasurableStat... stats) {
+    if (!isMetaSystemStore() || sensorName.equals("request")) {
```

Comment:
Why do we care about system store request? Is it for reading meta system store in DVC?

Reply:
stale code, reverted.